Classes/gitx_askpasswd_main.m

```diff
@@ -45,7 +45,7 @@ -(NSPanel*)	passwordPanel
 													  backing: NSBackingStoreBuffered defer: NO];
 		[mPasswordPanel setHidesOnDeactivate: NO];
 		[mPasswordPanel setLevel: NSFloatingWindowLevel];
-		[mPasswordPanel setTitle: @"GitX SSH Remote Login"];
+		[mPasswordPanel setTitle: NSLocalizedString(@"GitX SSH Remote Login", @"Title for password panel in command line tool")];
```

Comment:
Bonus points if anyone manages to embed `.strings` files in a plain executable 😆.